packages/nodes-base/nodes/GoToWebinar/descriptions/WebinarDescription.ts

```diff
@@ -406,6 +407,7 @@ export const webinarFields: INodeProperties[] = [
 			{
 				displayName: 'Is Password Protected',
 				name: 'isPasswordProtected',
+				// eslint-disable-next-line n8n-nodes-base/node-param-type-options-password-missing
```

Comment:
Edge case to cover later on, tracked [here](https://github.com/ivov/eslint-plugin-n8n-nodes-base/issues/168#issuecomment-1659876267).

Reply:
Ah just found another one of these further up